projects/ngneat/reactive-forms/src/lib/formGroup.spec.ts

```diff
@@ -33,6 +35,68 @@ const createGroup = (withError = false) => {
   );
 };
 
+const createArray = <T>(elements: T[], withError = false): FormArray<T> => {
+  const controlList = elements.map(element => new FormControl<T>(element)) as Array<AbstractControlOf<T>>;
+  return new FormArray<T>(controlList, withError ? errorFn : []);
+};
+
```

Comment:
Can you please add an advanced spec:

```ts
group = new FormGroup({
  a: new FormControl(),
  b: new FormGroup({
     c: new FormControl(),
     d: new FormControl()
  }),
  e: new FormGroup({
     f: new FormControl(),
     g: new FormControl()
  })
})


group.patchValue({
  b: {
    c: 'new' 
  },
  e: {
    g: 'new'
  }
})
```

Reply:
Exactly the same as code your code, or this just an example of .. how it should be implemented ?

Reply:
This is an example of one more spec I ask you to add, please